lecompteurdelaroulotte/app/src/main/res/layout/fragment_project.xml

```diff
@@ -96,7 +96,7 @@
         android:id="@+id/listCounters"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_marginTop="100dp"
+        android:layout_marginTop="108dp"
```

Comment:
That's actually an issue because it is not easy but made for little phones. Maybe we will have to change that when we will have multiple dimensions or we have to change it programmatically with Kotlin.

Reply:
Oh the value don't scale well different size of phones ? Perhaps we could use different layouts ? (or dynamically whatever fits)